backend/pyproject.toml

```diff
@@ -35,3 +35,45 @@ known_first_party = [
     "scale_down",
     "json_report"
 ]
+
+[tool.pyright]
+pythonVersion = "3.9"
+typeCheckingMode = "standard"
+include = [
+    "engine",
+    "lambdas",
+    "libs",
+    "utilities",
+]
+exclude = [
+    "**/.eggs",
+    "**/build",
+    "**/dist",
+]
+extraPaths = [
```

Comment:
Would this need to reference the github repo health check module at `engine/plugins/github_repo_health/lib`? Or would that be picked up through `engine`?

Reply:
I had the same thought!  I'll be honest, I don't know how Pyright is resolving the imports without explicitly adding it to the search paths, but it's somehow working.